### PR TITLE
Update meson, rebuild libpsl

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=meson crew update
```

### CODE REVIEW

File: packages/libpsl.rb [hunk 1]
```diff
@@ -3,19 +3,19 @@
 class Libpsl < Meson
   description 'C library for the Public Suffix List'
   homepage 'https://github.com/rockdaboot/libpsl'
-  @_ver = '0.21.2'
-  version @_ver.to_s
+  @_ver = '0.21.5'
+  version "#{@_ver}-1"
```
Comment (uberhacker) [inline packages/libpsl.rb:7]: How about just simply the following?
```
version '0.21.5'
...
git_hashtag version
```

Comment (satmandu) [reply packages/libpsl.rb:7]: This is replacing existing "0.21.5" binaries...

Comment (satmandu) [reply packages/libpsl.rb:7]: Ok that could work...

Comment (satmandu) [reply packages/libpsl.rb]: Done.

File: packages/mesonbuild.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 class Mesonbuild < Python
   description 'Meson is an open source build system meant to be both extremely fast and user friendly.'
   homepage 'https://mesonbuild.com/'
-  @_ver = '1.4.0'
+  @_ver = '1.5.0'
   version "#{@_ver}-py3.12"
```
Comment (uberhacker) [inline packages/mesonbuild.rb:7]:
```
version '1.5.0-py3.12'
...
git_hashtag version.split('-').first
```
?

Comment (satmandu) [reply packages/mesonbuild.rb]: Done.

File: .github/workflows/Unit-Test.yml [hunk 3]
```diff
@@ -57,7 +57,7 @@ jobs:
             ruby ../tests/commands/const.rb && \
             ruby ../tests/commands/help.rb && \
             ruby ../tests/commands/prop.rb && \
-            ( [[ -z "${ALL_CHANGED_FILES}" ]] && ( for file in ${ALL_CHANGED_FILES}; do
+            ( [[ -v ALL_CHANGED_FILES ]] && ( for file in ${ALL_CHANGED_FILES}; do
```
Comment (uberhacker) [inline .github/workflows/Unit-Test.yml:60]: Don't we want `[[ -n "${ALL_CHANGED_FILES}" ]] && ...`?  We're checking if there are files, correct?

Comment (satmandu) [reply .github/workflows/Unit-Test.yml:60]: The variable doesn't exist if it is empty.

Comment (uberhacker) [reply .github/workflows/Unit-Test.yml:60]:
```
    String operators:
    
      -z STRING      True if string is empty.
    
      -n STRING
         STRING      True if string is not empty.
```

Comment (uberhacker) [reply .github/workflows/Unit-Test.yml:60]: We can try your way but I know that `-n` definitely should work.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**